test/logged_smtp_delivery_test.rb

```diff
@@ -128,5 +128,26 @@ def without_file_logger
       _(mailer.response).must_be_kind_of Net::SMTP::Response
       _(mailer.response.status).must_equal '250'
     end
+
+    describe 'delivery failures' do
+      require 'net/smtp'
+      before do
+        class Net::SMTP
+          alias_method :orig_rcptto_list, :rcptto_list
+          def rcptto_list(*)
+            raise StandardError.new("kaboom")
+          end
+        end
```

Comment:
this honestly feels a little wierd. Why not use the FakeSMTP or stubbing to raise on this method call?

Reply:
It is admittedly a little weird, but I couldn't get the advertised minitest `Net::SMTP.mock :rcptto_list` call to work; it said there was no such method to mock.

Also, `FakeSMTP` is unused as of commit e5ad5c3ec86a141a1f684f883bc48b4fec071186 wherein the `adaptor` mechanism went away, and the fake `mailcrate` smtpd that this test uses isn't configurable enough to allow the error to come "naturally" from the SMTP transaction.